deploy/kubernetes/operator.antrea.vmware.com_v1_antreainstall_cr.yaml

```diff
@@ -79,12 +82,23 @@ spec:
     # Default MTU to use for the host gateway interface and the network interface of each Pod.
     # If omitted, antrea-agent will discover the MTU of the Node's primary interface and
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
-    #defaultMTU: 1450
+    #defaultMTU: 0
 
     # Whether or not to enable IPsec encryption of tunnel traffic. IPsec encryption is only supported
     # for the GRE tunnel type.
     #enableIPSecTunnel: false
 
+    # ClusterIP CIDR range for Services. It's required when AntreaProxy is not enabled, and should be
+    # set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver. When
+    # AntreaProxy is enabled, this parameter is not needed and will be ignored if provided.
+    #serviceCIDR: 10.96.0.0/12
```

Comment:
I don't think we need `serviceCIDR` in install CR, it is configured in cluster network CR

Reply:
> I don't think we need `serviceCIDR` in install CR, it is configured in cluster network CR

yes, for openshift we don't need it. But for kubernetes, we'll keep it as there is no cluster network CR, right?